src/layman/common/geoserver/output_srs_list_test.py

```diff
@@ -1,5 +1,63 @@
-from layman import settings
+import pytest
+from layman import settings, app
+from layman.util import url_for
+from test import process, process_client
+from layman.layer.geoserver.util import wms_direct, wfs_direct
+
+
+LAYERS_TO_DELETE_AFTER_TEST = []
+
+
+@pytest.fixture(scope="module")
+def app_config():
+    app.config['SERVER_NAME'] = settings.LAYMAN_SERVER_NAME
 
 
 def test_default_srs_list():
     assert settings.LAYMAN_OUTPUT_SRS_LIST == [4326, 3857]
+
+
+@pytest.fixture(scope="module")
+def delete_layer_after_test():
+    def register_layer_to_delete(workspace, layername):
+        LAYERS_TO_DELETE_AFTER_TEST.append((workspace, layername))
+    yield register_layer_to_delete
+    for workspace, layername in LAYERS_TO_DELETE_AFTER_TEST:
+        process_client.delete_layer(workspace, layername)
+
+
+@pytest.mark.usefixtures('app_config')
+@pytest.mark.xfail(reason="Not yet implemented")
+def test_custom_srs_list(delete_layer_after_test):
+    workspace = 'test_custom_srs_list_workspace'
+    layername = 'test_custom_srs_list_layer'
+    delete_layer_after_test(workspace, layername)
+    output_srs_list = [4326, 3857, 32633, 32634, 3059]
+    process.ensure_layman_function({
+        'LAYMAN_OUTPUT_SRS_LIST': ','.join([str(code) for code in output_srs_list])
+    })
+    process_client.publish_layer(workspace, layername)
+    assert_wms_output_srs_list(workspace, layername, output_srs_list)
```

Comment:
Add test, that when you first publish layer and change output_srs_list afterwards, it will works.